composer.json

```diff
@@ -25,12 +25,6 @@
     "support": {
         "issues": "https://github.com/zf1s/phpunit/issues"
     },
-    "repositories": [
-        {
-            "type": "pear",
-            "url": "https://pear.php.net"
-        }
-    ],
```

Comment:
let me get this straight, specifying this repository no longer works in composer v2?

Reply:
ok, I remember. It's been a long time. But I don't think this PR is really necessary, since it's specified in https://github.com/zf1s/phpunit/blob/master/.github/workflows/tests.yml#L52 to use composer v1, just offload the testing to github, why would you even want to run tests locally 😅

Reply:
@falkenhawk there have been too many issues to test it via github each time, new way works both in v1 and v2, so it's an improvement.

Reply:
yet it doesn't work on php 5.3, only the pear.php,net version is compatible

Reply:
@falkenhawk ok let it stay then.